ReactWindows/ReactNative/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -108,7 +104,7 @@ public override void LoadScript(IReactBridge bridge)
                 }
 
                 bridge.RunScript(_script, SourceUrl);
-                
+
                 _script = null;
```

Comment:
You can delete this last part nulling out the string, it was only done to try and get the JavaScript garbage collected (the paths will be quite small anyway)